scripts/ci/kubernetes/docker/bootstrap.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#  Licensed to the Apache Software Foundation (ASF) under one   *
+#  or more contributor license agreements.  See the NOTICE file *
+#  distributed with this work for additional information        *
+#  regarding copyright ownership.  The ASF licenses this file   *
+#  to you under the Apache License, Version 2.0 (the            *
+#  "License"); you may not use this file except in compliance   *
+#  with the License.  You may obtain a copy of the License at   *
+#                                                               *
+#    http://www.apache.org/licenses/LICENSE-2.0                 *
+#                                                               *
+#  Unless required by applicable law or agreed to in writing,   *
+#  software distributed under the License is distributed on an  *
+#  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+#  KIND, either express or implied.  See the License for the    *
+#  specific language governing permissions and limitations      *
+#  under the License.                                           *
+
+# launch the appropriate process
```

Comment:
Is this container just meant for testing or would this also become the recommended image?

It'd be nice to print something if neither of these arguments is being supplied. Could this maybe just directly call airflow, without this script?

Reply:
This would become the recommended image as we are able to call airflow with either "scheduler" or "webserver" as the arguments.

Is there any reason why we wouldn't want to offer the bootstrap script?

Reply:
I played around a bit with this container earlier and was confused for a minute when it just silently quit when I was trying to run it since I was missing these arguments.

All this script does is ignore all arguments apart from "webserver" and "scheduler", it'd be nicer to have at least a non-zero exit code when it does that, and really the script doesn't add anything?

Reply:
@ah- We need the bootstrap for the DB migrations when the webserver starts up, this is present in my changes in https://github.com/bloomberg/airflow/pull/46

Reply:
For CI that may be okay, but if this docker image is published @ah- is right.

For general use, for example, we don't want to call `initdb` everytime some container starts. That creates a ton of demo connections.

Reply:
@cbandy This is actually thing that many projects do, see https://github.com/pitkley/docker-netbox/blob/develop/entrypoint.sh#L82 for an example of netbox doing it. I would also expect the migrations to run if say, I upgraded the airflow version.

I've changed the bootstrap script in my branch so that it takes "$@" and passes that to airflow (https://github.com/bloomberg/airflow/pull/46/commits/fc8b30b69b53fedbce484edd6981cabb5ed697e8).